### PR TITLE
Box shadow looks better when it's translucent black rather than opaque grey.

### PR DESCRIPTION
**Pull request by breville**
before:

screenshot 2015-04-10 15 55 13

after:

screenshot 2015-04-10 16 56 48

note that this uses rgba() which isn't supported on IE8.  i could do a fallback to rgb() for solid shadows again, but if we're no longer supporting IE8 in any of these levels, seems simpler not to.

### CODE REVIEW

**Comment by breville** [comment]
This needs more testing to find each usage, but does the change look okay to folks?  I first noticed the issue with callouts.

**Comment by joshlory** [comment]
https://github.com/code-dot-org/code-dot-org/blob/0d2097ddb4c8a2433df24f0239d72265e3917c12/dashboard/public/apps-package/css/common.css#L96, L387 & 397 as well?

**Comment by joshlory** [comment]
Ah never mind, apps-package is the built CSS file.  The change in apps/style/common.scss will override it.

File: apps/style/common.scss [hunk 1]
```diff
@@ -614,7 +616,7 @@ input[type="radio"] {
   background-color: #fff;
   color: #000;
   border-radius: 6px;
-  box-shadow: 5px 5px 5px #888;
+  box-shadow: $shadow;
```
Comment (laurelfan) [inline apps/style/common.scss]: does this not need the dimensions?

**Comment by laurelfan** [comment]
lgtm :+1:

**Pull request merged**